### DISCUSSION TITLE
Is it possible to add checkboxes for filtering by Table column?

### DISCUSSION THREAD

**Discussion by tsarukyana** [filamentphp/filament#6509] [Help]
I need help when we are using **searchable(isIndividual: true)** It is adding only text input. But instead of it I want to add checkboxes and organize filtering functionality when selecting them.

`Tables\Columns\TextColumn::make('type')->searchable(isIndividual: true)`

How can we do that?

**Comment by danharrin** [comment]
You can create filters for that, the search filter is only for text

**Reply by tsarukyana** [reply]
@danharrin Ok, thanks. 
If we can use only Text input for current situation how to make send one query result when filling '120 - 150' value. It sends 5 requests instead of one. BUT for example '120-150'   is sending only one request? As you can see the difference is the spaces before and after "-".
